sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <div v-if="isSecondDisbursementAvailable" class="font-weight-bold my-3">
```

Comment:
pls, use the custom class. instead of bootstrap `font-weight-bold`

Reply:
why so? you meant to say I have to create a custom class which sets font weight to bold and use right?

Reply:
we already have classes built for it. use that

Reply:
![image](https://user-images.githubusercontent.com/77353155/191588486-b467c279-cfc5-4915-878f-d550df5eeefe.png)
![image](https://user-images.githubusercontent.com/77353155/191588651-64e5170e-a422-4a70-8569-d0adfbe9daeb.png)

Reply:
I tried to find one and couldn't find one (in base.scss). Can you please help me with the class name?

Reply:
you meant to say like .category-header-medium? I get it if so.